libraries/AP_Math/control.cpp

```diff
@@ -180,59 +492,33 @@ Vector2f sqrt_controller(const Vector2f& error, float p, float second_ord_lim, f
 // inverse of the sqrt controller.  calculates the input (aka error) to the sqrt_controller required to achieve a given output
 float inv_sqrt_controller(float output, float p, float D_max)
```

Comment:
We really should have some unit tests for this function.  We can probably test it against the sqrt_controller, after all.

Reply:
I don't know how to write ro run unit tests.